Makefile

```diff
@@ -1,5 +1,6 @@
 GO_MODULE := $(shell git config --get remote.origin.url | grep -o 'github\.com[:/][^.]*' | tr ':' '/')
-CMD_NAME := $(shell basename ${GO_MODULE})
+CMD_NAME := kanopy-codegen
+VERSION := $(shell grep '^const Version =' internal/version/version.go | cut -d\" -f2)
```

Comment:
nit: I'll add some official documentation to our styleguide to use ldflags to source the version into the build process

Reply:
Should this instead be included in https://github.com/10gen/kanopy-app-go?

Reply:
Yes.